examples/package.json

```diff
@@ -1,74 +1,63 @@
 {
   "name": "@cognite/reveal-examples",
-  "version": "0.1.0",
   "description": "Reveal examples",
-  "main": "dist/index.js",
-  "types": "dist/src/index.d.ts",
-  "files": [
-    "dist"
-  ],
+  "version": "0.1.0",
+  "private": true,
   "scripts": {
-    "release": "webpack",
-    "build": "webpack --env.development",
-    "clean": "rimraf -rf dist/ target/",
-    "serve": "webpack-dev-server --env.development --debug --color --progress --watch",
-    "test": "jest",
-    "coverage": "jest --coverage",
-    "lint": "tslint --project ."
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
+    "eject": "react-scripts eject"
   },
   "author": "Svenn-Arne Dragly <dragly@cognite.com>",
   "contributors": [
     "Lars Moastuen <lars.moastuen@cognite.com>",
     "Joachim Bjørne <joachim.bjorne@cognite.com>"
   ],
   "license": "Apache-2.0",
-  "glslify": {
-    "transform": [
-      "glslify-import"
-    ]
-  },
-  "devDependencies": {
-    "@cognite/tslint-config-cognite-3d": "^1.0.4",
-    "@tweenjs/tween.js": "^18.5.0",
-    "@types/dat.gui": "^0.7.5",
-    "@types/gl-matrix": "^2.4.5",
-    "@types/jest": "^25.2.3",
-    "@types/node": "^14.0.5",
-    "@types/tween.js": "^18.5.1",
-    "copy-webpack-plugin": "^5.1.1",
-    "css-loader": "^3.5.3",
-    "file-loader": "^6.0.0",
-    "html-webpack-plugin": "4.3.0",
-    "prettier": "^1.19.1",
-    "raw-loader": "^4.0.1",
-    "rimraf": "^3.0.2",
-    "source-map-loader": "^0.2.4",
-    "ts-loader": "7.0.4",
-    "tslint": "^5.20.1",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-no-circular-imports": "^0.7.0",
-    "tslint-plugin-prettier": "^2.3.0",
-    "typescript": "3.9.3",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.11.0",
-    "webpack-log": "^3.0.1",
-    "webpack-node-externals": "^1.7.2",
-    "worker-plugin": "^4.0.3"
-  },
   "dependencies": {
-    "@cognite/potree-core": "^1.1.3",
     "@cognite/reveal": "../viewer/dist",
     "@cognite/sdk": "^2.28.0",
-    "@xailabs/three-renderer-stats": "^1.0.4",
     "camera-controls": "^1.21.0",
-    "comlink": "^4.3.0",
     "dat.gui": "^0.7.7",
-    "gl-matrix": "^3.3.0",
-    "glslify": "^7.0.0",
-    "hold-event": "0.0.1",
-    "postprocessing": "^6.13.5",
-    "rxjs": "^6.5.5",
+    "hold-event": "^0.0.1",
```

Comment:
What has happened for the dependencies to have changed this much?

Reply:
I recreated examples from scratch. So some dependencies are not needed anymore. Some was unused (rxjs), those related to build are now part of react-scripts and they are not exposed in our package.json